pkgs/top-level/all-packages.nix

```
@@ -4828,6 +4828,10 @@ in
 
   kippo = callPackage ../servers/kippo { };
 
+  kimg = callPackage ../development/tools/misc/kimg {
+    asciidoc = asciidoc-full;
+  };
+
```

Any reason to use `imagemagick7` as opposed to `imagemagick7Big`? At any rate, it is better to use `imagemagick` as an argument and then set this attribute to `imagemagick7Big` at the `callPackage` site:

```suggestion
  kimg = callPackage ../development/tools/misc/kimg {
    asciidoc = asciidoc-full;
    imagemagick = imagemagick7Big;
  };
```

This allows users to override the derivation in a more straightforward manner. E.g.

```nix
kimg.override { imagemagick = imagemagick7; }
```

as opposed to:

```nix
kimg.override { imagemagick7Big = imagemagick7; }
```